daily-challenges/2024-09/17/huynhtram1504.spec.ts

```
@@ -0,0 +1,95 @@
+    await expect(page.locator("//tbody[@id='cart-items']/tr[1]/td[3]")).toHaveText("2");
+    await expect(page.locator("//tbody[@id='cart-items']/tr[2]/td[3]")).toHaveText("2");
+    await expect(page.locator("//tbody[@id='cart-items']/tr[3]/td[3]")).toHaveText("3");
+
+    //(Nâng cao) Kiểm tra tổng tiền sản phẩm đúng (tổng tiền = tổng (số lượng * đơn giá))
+    let totalMoney = 0;
+    product.forEach(item => totalMoney += item.price * item.quantity);
+    //Retreives all matching rows into an array
+    const rowLocators = await page.locator("//tbody//tr").all();
+    //For each row, locate all its cells(<td> elements)
+    //Store these cells in an array
+    let cartTotal = 0;
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)